oauth2/oauth2-resource-server/src/main/java/org/springframework/security/oauth2/server/resource/authentication/JwtAuthenticationConverter.java

```diff
@@ -65,4 +73,14 @@ public void setJwtGrantedAuthoritiesConverter(Converter<Jwt, Collection<GrantedA
 		Assert.notNull(jwtGrantedAuthoritiesConverter, "jwtGrantedAuthoritiesConverter cannot be null");
 		this.jwtGrantedAuthoritiesConverter = jwtGrantedAuthoritiesConverter;
 	}
+
+	/**
+	 * Sets the principal claim name.
+	 *
+	 * @param principalClaimName The principal claim name
+	 * @since 5.2
```

Comment:
Since this will go out in 5.4, would you change the `@since` to reflect that?

Reply:
Done.